phpBB/phpbb/extension/extension_interface.php

```diff
@@ -27,6 +27,13 @@
 	public function is_enableable();
 
 	/**
+	* Indicate why the extension can not be enabled.
+	*
+	* @return string|array with actual processed language string(s) (not language keys), or empty
+	*/
+	public function get_not_enableable_msg();
```

Comment:
I agree with you: the name is terribly ugly... but yours is not much better 😄.  I prefer `not_enableable_msg` if you don't mind, and yes, I will change it.

Regarding the CLI, I was under the impression that this was not considered relevant there.  In fact, the `is_enableable` method is not called (directly) at all in the CLI, it calls `enable` directly (also without going step by step).  If you think this is benefitial, I am happy to add support for it in the CLI, no problem.